src/nylo/tokens/symbol.py

```diff
@@ -1,37 +1,60 @@
+# This file is a part of nylo
```

Comment:
symbol.py should not have the copyright. Only file beggining with __ shall have them.